library/build.gradle

```
@@ -1,15 +1,17 @@
-import java.nio.charset.StandardCharsets
+import java.nio.charset.StandardCharsets // Keep if getAppName or other parts need it
```

Redundant comment. Makes the commit noisy.
```suggestion
import java.nio.charset.StandardCharsets
```